pkg/interface/chat/src/js/components/join.js

```diff
@@ -2,59 +2,37 @@ import React, { Component } from 'react';
 import classnames from 'classnames';
 
 
-export class LandingScreen extends Component {
+export class JoinScreen extends Component {
 
   constructor(props) {
     super(props);
   }
 
   componentDidMount() {
     const { props } = this;
-    let station = props.match.params.ship + '/' + props.match.params.station;
+    let station = `${props.match.params.ship}/${props.match.params.station}`;
 
-    if (station in props.configs) {
-      props.history.push(`/~chat/${station}`);
+    if (station in props.inbox) {
+      props.history.push(`/~chat/room${station}`);
```

Comment:
Should this be `/~chat/room/${station}`?  Or alternatively should `station` be prefixed with a `/` (as it is below)?

Reply:
(Alternatively, maybe `props.match.params.ship` contains a prefixing `/`, but this seems error-prone.)

Reply:
Yeah fair enough.  This particular one may still be missing the prefix, though; note that `station` here is defined via:

```
    let station = `${props.match.params.ship}/${props.match.params.station}`;

    if (station in props.inbox) {
      props.history.push(`/~chat/room${station}`);
    }
```

but in `components/chat.js`, `station` has a manual `/` prefix:

```
station: `/${props.match.params.ship}/${props.match.params.station}`,
```

Reply:
Good catch. There was indeed an error here.

Reply:
Yeah, this one in particular seems to be an error